standalone/src/models/organization/organization/team/fields-config.ts

```diff
@@ -0,0 +1,25 @@
+import { fake, type TModelFieldsConfig } from '@/core';
+import { ReferenceRest, ReferenceGraphql } from '@/models/commons/reference';
+import { User } from '@/models/user';
+import type { TTeamGraphql, TTeamRest } from './types';
+
+const commonFieldsConfig = {
+  id: fake((f) => f.string.uuid()),
+  name: fake((f) => f.commerce.department()),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TTeamRest> = {
+  fields: {
+    ...commonFieldsConfig,
+    members: fake(() => [ReferenceRest.random().typeId('user')]),
+  },
+};
+
+export const graphqlFieldsConfig: TModelFieldsConfig<TTeamGraphql> = {
+  fields: {
+    ...commonFieldsConfig,
+    __typename: 'Team',
+    members: [User.random().buildGraphql()],
+    membersRef: [ReferenceGraphql.random().typeId('user').build()],
```

Comment:
FYI, with the new pattern, `postbuild` function of fakerjs is run after nested models are generated. That means, you can relatively comfortably sync the id of the generated User, with the user ref inside membersRef, so that overall generated data is internally consistent.

https://github.com/commercetools/test-data/blob/dc773044cd54b27861b13c702fca570e3d03829f/docs/guidelines/creating-new-model.md?plain=1#L120-L126